src/errors.ts

```diff
@@ -0,0 +1,54 @@
+// String values used in user-facing error messages
```

Comment:
As an optional suggestion in this vein, we've quite liked having a `{success: boolean, message?: string}` `GenericResponse` type api endpoints to base return-types on.